src/components/PeopleTable.tsx

```diff
@@ -1,5 +1,40 @@
+import React from 'react';
+import { Person } from '../types';
+import { useParams, useSearchParams } from 'react-router-dom';
+import classNames from 'classnames';
+import PersonLink from './PersonLink';
+import { SearchLink } from './SearchLink';
+
+type Props = {
+  sortedPeople: Person[];
+  getSortIcon: (sorted: string) => string;
+  peoples: Person[];
+};
+
 /* eslint-disable jsx-a11y/control-has-associated-label */
-export const PeopleTable = () => {
+export const PeopleTable: React.FC<Props> = ({
+  sortedPeople,
+  getSortIcon,
+  peoples,
+}) => {
+  const [searchParams] = useSearchParams();
+  const { peopleId } = useParams();
```

Comment:
```suggestion
  const { personId } = useParams();
```
"people" is plural form